setup.cfg

```diff
@@ -11,5 +11,15 @@ filterwarnings =
     # Warnings that statsmodels raises a lot of
     ignore::statsmodels.tools.sm_exceptions.HessianInversionWarning
 
+    # statsmodels warning that doesn't apply to us
+    ignore:fft=True
+
+    # joblib warning that we can't control (fixed in this commit https://github.com/joblib/joblib/commit/a861f43167ab63fe683c45679e34143751cb976d, but not deployed yet)
+    # This is the solution from pytest: https://github.com/pytest-dev/pytest/issues/4116#issuecomment-429101898
+    ignore:tostring.*is deprecated
```

Comment:
Hopefully this doesn't mask something that later bites us. 😬

Reply:
I subscribed to release notifications from joblib to remember to remove it in the future